docs/fundamentals/networking/sockets/tcp-classes.md

```diff
@@ -78,10 +78,203 @@ The preceding C# code:
 - Writes the sent message to the console.
 - Finally, calls the <xref:System.Net.Sockets.TcpListener.Stop%2A> method to stop listening on the port.
 
+## Advanced TCP control with `Socket` class
+
+`TcpListener` and `TcpClient` classes internal implementation relies on the `Socket` class. This means you can do everything with `Socket` that you can do with `TcpClient` and `TcpListener`.
+
+### Create a client
+
+`TcpClient`'s default constructor tries to create a [_dual-stack socket_](<xref:System.Net.Sockets.Socket.DualMode>) via `[new Socket(SocketType, ProtocolType)](xref:System.Net.Sockets.Socket)` constructor, if [IPv6 is supported (Socket.OSSupportsIPv6 == true)](<xref:System.Net.Sockets.Socket.OSSupportsIPv6>). Otherwise, it fallbacks to IPv4 socket. Consider the following TCP client code:
+
+```csharp
+var client = new TcpClient();
+```
+
+The preceding TCP client code is functionally equivalent to the following socket code:
+
+```csharp
+var socket = new Socket(SocketType.Stream, ProtocolType.Tcp);
+```
+
+One of `TcpClient`'s constructor takes `AddressFamily` enum as a parameter. You can pass three different enum values to this constructor, otherwise, it throws an <xref:System.ArgumentException>. Valid enums are:
```

Comment:
Need to link the specific constructor.

Reply:
IIUC, it's not possible to link to `specific` overload, but I changed it into `constructors` link. 

> Need to link the specific constructor.
> 

@IEvangelist is it possible?

Reply:
Yes, you can link to a specific `.ctor`. Are you using the Learn Authoring Pack?

Reply:
Yes, I wouldn't be able to find a step to `link to specific`.